Test2/Tests/HistoryPageTests.cs

```diff
@@ -0,0 +1,145 @@
+                }
+
+                tabelValues.Add(RowValues);
+            }
+        }
+
+        [Test]
+        public void List2()
+        {
+            new HistoryPage(driver).Open();
+            List<List<string>> tabelValues = new List<List<string>>();
+            int RowsCount = driver.FindElements(By.XPath("//tr")).Count();
+
+            for (int rowNumber = 0; rowNumber < RowsCount; rowNumber++)
+            {
+                List<string> rowValues = new List<string>();
+                for (int colNumber = 0; colNumber < 8; colNumber++)
+                {
+                    IWebElement cell = driver.FindElement(By.XPath($"//tr[{rowNumber+1}]//*[{colNumber+1}]"));
+                    string cellText = cell.Text;
+                    rowValues.Add(cellText);
+                }
+                tabelValues.Add(rowValues);
+            }
+        }
+
+        [Test]
+        public void CheckElements()
+        {
+            new HistoryPage(driver).Open();
+            HistoryPage historyPage = new HistoryPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+            List<List<string>> expected2 = new List<List<string>>() {
+                new List<string>() { "Amount", "%", "Term", "Year", "From", "To", "Interest", "Income" },
+                new List<string>() { "1,000.00", "50%", "360", "365", "01/01/2022", "27/12/2022", "493.15", "1,493.15" },
+                new List<string>() { "1,000.00", "50%", "360", "365", "01/01/2022", "27/12/2022", "493.15", "1,493.15" },
+                new List<string>() { "1,000.00", "50%", "360", "365", "01/01/2022", "27/12/2022", "493.15", "1,493.15" },
+                new List<string>() { "1,000.00", "50%", "360", "365", "01/01/2022", "27/12/2022", "493.15", "1,493.15" },
+                new List<string>() { "1,000.00", "50%", "360", "365", "01/01/2022", "27/12/2022", "493.15", "1,493.15" },
+                new List<string>() { "200.00", "100%", "365", "365", "01/01/2022", "01/01/2023", "200.00", "400.00" },
+                new List<string>() { "200.00", "100%", "365", "365", "01/01/2022", "01/01/2023", "200.00", "400.00" },
+                new List<string>() { "200.00", "100%", "365", "365", "01/01/2022", "01/01/2023", "200.00", "400.00" },
+                new List<string>() { "200.00", "100%", "365", "365", "01/01/2022", "01/01/2023", "200.00", "400.00" },
+                new List<string>() { "200.00", "100%", "365", "365", "01/01/2022", "01/01/2023", "200.00", "400.00" },
+            };
+
+            historyPage.ClearButton.Click();
+            for (int i = 0; i < 5; i++)
+            {
+                calculatorPage.Open();
+                calculatorPage.Calculate("200", "100", "365", "01", "January", "2022");
+            }
+            for (int i = 0; i < 5; i++)
+            {
+                calculatorPage.Open();
+                calculatorPage.Calculate("1000", "50", "360", "01", "January", "2022");
+            }
```

Comment:
You can open the `calculatorPage` only once:

```suggestion
            historyPage.ClearButton.Click();
            calculatorPage.Open();
            for (int i = 0; i < 5; i++)
            {
                calculatorPage.Calculate("200", "100", "365", "01", "January", "2022");
            }
            for (int i = 0; i < 5; i++)
            {
                calculatorPage.Calculate("1000", "50", "360", "01", "January", "2022");
            }
```